releases/release-6.2.0.md

```diff
@@ -0,0 +1,450 @@
+
+    [User document](/system-variables.md#tidb_default_string_match_selectivity-new-in-v620) [#36209](https://github.com/pingcap/tidb/issues/36209) @[time-and-fate](https://github.com/time-and-fate)
+
+* Window functions pushed down to TiFlash can be executed in multiple threads
+
+    By enabling the Fine Grained Shuffle feature, the window function can be executed from in a single thread to in multiple threads. It reduces the query response time significantly. This feature does not change user behavior. You can control the granularity of the shuffle by adjusting the value of the variables.
+
+    [User document](/system-variables.md#tiflash_fine_grained_shuffle_batch_size-new-in-v620) [#4631](https://github.com/pingcap/tiflash/issues/4631) @[guo-shaoge](https://github.com/guo-shaoge)
+
+* TiFlash supports a newer version of storage format
+
+    The new storage format relieves high CPU usage caused by GC in high-concurrency and heavy workload scenarios. This significantly reduces IO traffic of background tasks, thereby boosting stability under high concurrencies and heavy workloads. At the same time, space amplification and disk waste can be significantly reduced.
+
+    In TiDB v6.2.0, data is stored in the new storage format by default. Note that if TiFlash is upgraded from earlier versions to v6.2.0, you cannot perform in-place downgrade on TiFlash, because earlier TiFlash versions cannot recognize the new storage format.
+
+    For more information about upgrading TiFlash, see [TiFlash v6.2.0 Upgrade Guide](/tiflash-620-upgrade.md).
+
+    [User document](/tiflash/tiflash-configuration.md#configure-the-tiflashtoml-file) [#3594](https://github.com/pingcap/tiflash/issues/3594) @[JaySon-Huang](https://github.com/JaySon-Huang) @[jiaqizho](https://github.com/jiaqizho) @[lidezhu](https://github.com/lidezhu)
+
+* TiFlash optimizes data scanning performance in multiple concurrency scenarios (experimental)
+
+    TiFlash reduces duplicate reads of the same data by merging read operations of the same data, and optimizes the resource overhead in the case of multiple concurrent tasks to improve data scanning performance. It avoids the situation where the same data has to be read separately in each task, or even the same data may be read multiple times at the same time, if the same data is involved in multiple concurrent tasks.
+
+    [User document](/tiflash/tiflash-configuration.md#configure-the-tiflash-toml-file) [#5376](https://github.com/pingcap/tiflash/issues/5376) @[JinheLin](https://github.com/JinheLin)
+
+* TiFlash adds Fast Mode for data scanning to increase read and write speed by sacrificing data consistency (experimental)
```

Comment:
```suggestion
* TiFlash adds FastScan for data scanning to increase read and write speed by sacrificing data consistency (experimental)
```

Reply:
```suggestion
* TiFlash adds Fast Mode for data scanning to increase read speed by sacrificing data consistency (experimental)
```

Reply:
only for read, no affects to write speed